modules/k3s/k3s.go

```diff
@@ -163,3 +164,21 @@ func unmarshal(bytes []byte) (*KubeConfigValue, error) {
 	}
 	return &kubeConfig, nil
 }
+
+// LoadImages loads images into the k3s container.
+func (c *K3sContainer) LoadImages(ctx context.Context, images string) error {
+	imageFile := filepath.Base(images)
+	containerPath := fmt.Sprintf("/tmp/%s", imageFile)
+
+	err := c.Container.CopyFileToContainer(ctx, images, containerPath, 0x644)
+	if err != nil {
+		return fmt.Errorf("copying image to container %w", err)
+	}
+
+	_, _, err = c.Container.Exec(ctx, []string{"ctr", "-n=k8s.io", "images", "import", containerPath})
+	if err != nil {
+		return fmt.Errorf("importing image %w", err)
+	}
+
+	return nil
+}
```

Comment:
You probably need a new line here

Reply:
Interesting, the linter didn't point out this error :thinking:

Reply:
With gofumpt/goflt you need a new line at the end of the file. But for a reason I ignore gci is the one saying something is wrong . I suspect that gci is trying to apply gofmt plus the import ordering rules